### DISCUSSION TITLE
2024-07-24 已暂停答题

### DISCUSSION THREAD

**Discussion by kaedea** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#209] [General]
![Screenshot_2024-07-24-08-02-40-114_com.xiaomi.vipaccount.jpg](https://github.com/user-attachments/assets/987c3953-cd2e-498b-b41e-f19c6a7091a9)

这什么情况...

**Comment by xbn2002** [comment]
改明天了，耍猴呢
